packages/bpk-component-map/src/BpkMap.js

```diff
@@ -82,29 +106,37 @@ const BpkMap = withGoogleMap((props: Props) => {
     gestureHandling = 'greedy';
   }
 
+  const ref = useRef(null);
+  const mapContainerClassName = getClassName('bpk-map', className);
```

Comment:
This property will style the map's div container, defaulting to a height of 100% to render it fully.
You can set the container style directly here through className, or you can wrap another layer of div in the outer layer to specify the style like in the example.js.